### DISCUSSION TITLE
page index gets reset on fetching data from server

### DISCUSSION THREAD

**Discussion by ronaksonigara** [TanStack/table#1985] [General]
https://github.com/tannerlinsley/react-table/blob/master/examples/pagination-controlled/README.md

took refrence from above link but page index gets reset on pagination

**Comment by vsSanti** [comment]
There is an explanation at this link. I think it would solve your problem.
https://github.com/tannerlinsley/react-table/blob/master/docs/faq.md

**Comment by ronaksonigara** [comment]
@vsSanti thanks

**Comment by onenick972** [comment]
https://codesandbox.io/s/cool-dawn-ecut7

I created a sandbox with server-side pagination and sorting..the index issue is also fixed
table has material-UI
